plugins/modules/system_access_users.py

```diff
@@ -241,12 +241,11 @@ def main():
                 if ansible_user.apikeys:
                     result["generated_apikeys"] = []
```

Comment:
Why not just get the list from the User?
```suggestion
                    result["generated_apikeys"] = ansible_user.apikeys
```